Adjust calculate-activity-size

sfmskywalker opened the pull request:
Adjust calculate-activity-size to ensure measurement is accurate with new class selector and fix comments for readability.

Fixes #237

sfmskywalker committed:
Refactor badge rendering in ActivityWrapper

Consolidate badge rendering logic to simplify code and handle null Stats more effectively. Adjust calculate-activity-size to ensure measurement is accurate with new class selector and fix comments for readability.

raymonddenhaan commented on src/modules/Elsa.Studio.Workflows.Designer/Components/ActivityWrapper.razor:
```diff
@@ -3,20 +3,15 @@
-        var color = Stats.Faulted ? Color.Error : Stats.Blocked ? Color.Warning : Stats.Uncompleted > Stats.Completed ? Color.Info : Stats.Completed > 0 ? Color.Success : Color.Default;
-        var content = Stats.Blocked ? null : Stats.Completed.ToString();
-        var icon = content == null ? Stats.Blocked ? Icons.Material.Outlined.HourglassTop : Stats.Uncompleted > 0 ? Icons.Material.Outlined.PlayArrow : null : null;
+    
```
Are you deliberately reverting this change or is this a merge issue?

sfmskywalker replied:
this is deliberate; originally I fixed the height issue by reverting the rendering optimization. Now, I am applying the render optimization and fix the height issue by updating the JS file that calculates the activity element's height.

sfmskywalker committed:
Refactor `ActivityWrapper.razor` layout structure.

Combine `DisplayActivityWrapper` method into main markup for readability and maintainability. Simplified inline styles and constants initialization to improve code clarity.
